doc/api/fs.md

```diff
@@ -2529,6 +2529,19 @@ changes:
 Asynchronous rename(2). No arguments other than a possible exception are given
 to the completion callback.
 
+This function will take the old path name of a file and rename it to the path
+name provided in the second parameter.
+
+Here is an example below (assuming the file "firstfile.txt" already exists):
```

Comment:
No need to say "Here is an example below..." Just provide the example. If you want it to be clear what the example does, add a code comment to the example along the lines of:

```js
// Renames firstfile.txt to secondfile.txt
```